src/Root.js

```diff
@@ -12,12 +13,19 @@ export default class ReduxRoot {
   };
 
   constructor(props) {
-    this.dispatcher = createDispatcher();
-    this.dispatcher.receiveStores(props.stores);
+    this.dispatcher = props.dispatcher || createDispatcher();
```

Comment:
It isn't good for two reasons:
* Dispatcher from defaultProps will be shared between all Root instances.
* It will be created even if we don't use it.